capsules/src/process_console.rs

```diff
@@ -989,3 +1092,6 @@ impl<'a, A: Alarm<'a>, C: ProcessManagementCapability> uart::ReceiveClient
         let _ = self.uart.receive_buffer(read_buf, 1);
     }
 }
+
+/*
```

Comment:
What is the purpose of these comment lines?

Reply:
None; simply forgot to remove them. Should be gone by now